.golangci.yml

```diff
@@ -33,7 +33,6 @@ linters:
     staticcheck:
       checks:
         - all
-        - '-ST1003' # disable rule 'Poorly chosen identifier'
```

Comment:
note: This is  the main change. I enabled this rule and then updates all linting errors.